Lab 1/Lab1.ipynb

```diff
@@ -2,11 +2,539 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(0,40,41)\n",
+    "pmf = stats.binom.pmf\n",
+    "plt.plot(x, pmf(x, n = 40, p = 0.5))\n",
```

Comment:
Try using plt.stairs() or plt.step() here to show more clearly that this is a plot of a discrete variable. You could also use plt.vlines([x-locations], ymin, ymax) I.E. plt.vlines(x, 0, pmf).

Do this for the next plot as well.